save-demo-agent/src/nativeMain/kotlin/com/saveourtool/save/demo/agent/Main.kt

```diff
@@ -0,0 +1,42 @@
+/**
+ * An entry point for save-demo-agent.
+ */
+
+package com.saveourtool.save.demo.agent
+
+import com.saveourtool.save.core.utils.ProcessBuilder
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.cio.*
+import io.ktor.server.engine.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import okio.FileSystem
+
+@Suppress("MagicNumber")
+fun main() {
+    runServerOnPort(23456).start()
+}
+
+/**
+ * @param port port number
+ * @return [CIOApplicationEngine]
+ */
+@Suppress("MagicNumber")
+internal fun runServerOnPort(port: Int) = ProcessBuilder(true, FileSystem.SYSTEM).let { pb ->
```

Comment:
why do we need processbuilder for running server? we can't do it directly via ktor?

Reply:
We use process builder only to run shell command for now.